guides/common/modules/proc_creating-variables-for-convert2rhel-role.adoc

```diff
@@ -13,14 +13,20 @@ You must create the following additional variables:
 | {project-context}_server_url | string | URL of {project-context} instance
 | {project-context}_username | string | admin
 | {project-context}_validate_certs |boolean | do ansible cert checks?
+| {project-context}_content_rhel_enable_rhel7 | boolean | true
+| {project-context}_content_rhel_enable_rhel8 | boolean | true
```

Comment:
I am not sure if we should say that the default value is `true` if that depends on the type of the conversion. I would suggest `true if converting to RHEL 7` and `true if converting to RHEL 8` instead. However, this will need a review from someone else to confirm.

Reply:
@adamlazik1 default value is not going to change in either way. Users should update the value based on which version they are converting to. So I guess default value column should have no change.

Reply:
@maximiliankolb a gentle reminder!

Reply:
I don't understand the third column. Is it the default value? Then the title should be shortened to "Default value".

I'd rather like to see a short description; optionally the default value in brackets, for example:


```diff
-| {project-context}_validate_certs |boolean | do ansible cert checks?
+| {project-context}_validate_certs |boolean | Enable certificate check in Ansible (default: true)
```

In general, I lean towards @adamlazik1's idea of adding "true if converting to RHEL 7". I assume you'd only want to provide either EL7 or EL8 content.